Applications/Games/Anno 2070/Uplay/script.js

```diff
@@ -0,0 +1,26 @@
+include(["engines", "wine", "quick_script", "uplay_script"]);
+include(["engines", "wine", "verbs", "corefonts"]);
+include(["engines", "wine", "plugins", "virtual_desktop"]);
+
+var installerImplementation = {
+    run: function () {
+        new UplayScript()
+            .name("Anno 2070")
+            .editor("Ubisoft")
+            .applicationHomepage("http://anno-game.ubi.com/anno-2070/en-US/")
+            .author("Zemogiter")
+            .wineVersion("3.20")
+            .wineDistribution("upstream")
+            .appId(22)
+            .preInstall(function (wine, wizard){
+                wizard.message(tr("Please install winbind before installing Uplay."));
```

Comment:
Is winbind always required for Uplay? If so, this should be added to `UplayScript` and not here.

Reply:
AppDB results state this package (so it have to be installed manually by the user via Synaptic or .deb file or other method) is needed to get rid of warnings when starting Uplay.

Reply:
In this case, this should really be implemented in the `UplayScript`.